projects/igniteui-angular/src/lib/buttonGroup/buttonGroup.component.ts

```diff
@@ -119,12 +119,12 @@ export class IgxButtonGroupComponent extends DisplayDensityBase implements After
     }
 
     /**
-     * An @Input property that enables selecting multiple buttons. By default, multi-selection is false.
+     * An @Input property that sets the selection mode of the buttons. By default, the selection mode is single.
      * ```html
-     * <igx-buttongroup [multiSelection]="false" [alignment]="alignment"></igx-buttongroup>
+     * <igx-buttongroup [selectionMode]="'multi'" [alignment]="alignment"></igx-buttongroup>
      * ```
      */
-    @Input() public multiSelection = false;
+    @Input() public selectionMode: 'single' | 'singleRequired' | 'multi' = 'single';
```

Comment:
@georgianastasov @rkaraivanov 
The PR says `multiSelection` is "deprecated", but this essentially replaces it with the new prop, which results in an actual breaking change, which I don't think was intended. Can we keep the old prop as deprecated and proxy get/set of the new `selectionMode`?

Reply:
There is a new PR [here](https://github.com/IgniteUI/igniteui-angular/pull/13521) with the new required changes.